static/css/nl_interface.scss

```diff
@@ -237,6 +237,10 @@ $search-section-bottom-height: 7rem;
       height: 100%;
     }
   }
+
+  .subtopics-section a {
```

Comment:
could you add this to the group in line 256

Reply:
done